src/System.Linq/src/System/Linq/Set.cs

```diff
@@ -191,6 +191,27 @@ internal List<TElement> ToList()
         }
 
         /// <summary>
+        /// Creates a hashset from the items in this set.
+        /// </summary>
+        /// <returns>A list of the items in this set.</returns>
+        internal HashSet<TElement> ToHashSet(IEqualityComparer<TElement> comparer)
```

Comment:
Nit: Please add an XML doc for the comparer before the `returns` tag. `<param name="comparer">The equality comparer for the new <see cref="HashSet{T}"/>.</param>`